server/service/client_scripts.go

```diff
@@ -36,6 +37,14 @@ func (c *Client) RunHostScriptSync(hostID uint, scriptContents []byte) (*fleet.H
 			return nil, fmt.Errorf("decoding %s %s response: %w, body: %s", verb, path, err, b)
 		}
 	case http.StatusForbidden:
+		b, err := io.ReadAll(res.Body)
```

Comment:
Maybe it's cleaner to use `extractServerErrMsg`? (You will need to compare strings anyway... so maybe not...)

Reply:
Yep, that's cleaner. Switched to using that instead.